community/steam_input/steam_input_brazilian.vdf

```diff
@@ -9,37 +9,37 @@
 "[english]Button_SecondaryAttack"		"Secondary Attack"
 "Button_MeleeAttack"		"Corpo a corpo"
 "[english]Button_MeleeAttack"		"Melee"
-"Button_OffHandAttack"		"Usar item improvisado"
+"Button_OffHandAttack"		"Usar item secundário"
```

Comment:
Marines have three equipment slots: primary, secondary, and extra. This button activates the extra slot.

Reply:
@BenLubar which word could be used as a synonym to "off-hand" if you had to change it?

Reply:
I had a similiar situation in my german translation. At some point I settled on one word which translates back well too.
Maybe you can find an inspiration for these back-translations into english:
https://dict.leo.org/german-english/Zusatzausr%C3%BCstung

Reply:
> How do you feel about using "item adicional" ("additional item") as an alternative for "offhand item"?

That sounds good to me.